frontend/app/src/App.tsx

```diff
@@ -1617,7 +1603,6 @@ export class App extends PureComponent<Props, State> {
   clearCache = (): void => {
     this.closeDialog()
     if (this.isServerConnected()) {
-      this.metricsMgr.enqueue("clearCache")
```

Comment:
Will this address the hotkey for clearCache, or are we not worried about it?

Reply:
Good catch! Hotkey usage won't be covered by the menu click event. But I afaik (and based on the data team feedback) the `clearCache` event hasn't been used for anything yet and might also be a bit more complicated to get any useful insights. In comparison, the menu clicks can just be compared to other menu click events.

Tracking hokey usage could be a valid aspect, but I think it would have more value to cover all hotkeys under a `hotkey` event similar to menu clicks. 

I will check back with data team if they have any thoughts on the hotkey aspect.